docs/src/components/DocsPageWrapper/DocsPageWrapper.js

```diff
@@ -91,6 +93,14 @@ const PlatformNav = ({
     >
       iOS
     </BpkHorizontalNavItem>
+    <BpkHorizontalNavItem
+      name="swiftui"
+      disabled={disableSwiftUITab}
+      selected={platform === 'swiftui'}
+      onClick={onSwiftUIClick}
+    >
+      iOS
```

Comment:
should this be called SwiftUI? (maybe also include preview - we did that for compose originally)

Reply:
good catch!